app/actions/wallets-actions.js

```diff
@@ -4,7 +4,7 @@ import Action from './lib/Action';
 // ======= WALLET ACTIONS =======
 
 export default class WalletsActions {
-  createWallet: Action<{ name: string, currency: string }> = new Action();
+  createWallet: Action<{ name: string, currency: string, password: string }> = new Action();
```

Comment:
I was thinking about that but since we default the password to empty string here: https://github.com/input-output-hk/daedalus/blob/feature/daef-168-optionally-setting-a-password-during-wallet-creation/app/components/wallet/WalletCreateDialog.js#L130 it should never happen that it has null value but if you want I can change it to `?string`